docs/advanced-guides/firehose.mdx

```diff
@@ -71,3 +70,29 @@ limited concurrency based on who the event is for.
 Once we have a scheduler, we call into `HandleRepoStream` which does the actual
 decoding of the data coming over the websocket and calls into the event handler
 we wrote.
+
+If you want a more simplified firehose where events are JSON, you can connect
+to the [JetStream](https://jazco.dev/2024/09/24/jetstream/) websocket:
```

Comment:
could you have this link go to https://docs.bsky.app/blog/jetstream? it has updated context

Reply:
Great point! Added.